source/renderer/app/api/utils/localStorage.js

```diff
@@ -7,7 +7,8 @@ const networkForLocalStorage = String(environment.NETWORK);
 const storageKeys = {
   USER_LOCALE: networkForLocalStorage + '-USER-LOCALE',
   TERMS_OF_USE_ACCEPTANCE: networkForLocalStorage + '-TERMS-OF-USE-ACCEPTANCE',
-  THEME: networkForLocalStorage + '-THEME'
+  THEME: networkForLocalStorage + '-THEME',
+  DATA_LAYER_MIGRATION_ACCEPTANCE: networkForLocalStorage + '-DATA_LAYER_MIGRATION_ACCEPTANCE',
```

Comment:
@daniloprates can you please change the last part of this line so that the underlines are replaced with dashes? 
`-DATA_LAYER_MIGRATION_ACCEPTANCE` -> `-DATA-LAYER-MIGRATION-ACCEPTANCE`
...the key should remain as is!